florence-core/src/florence/core/model/shared/FontSize.scala

```diff
@@ -0,0 +1,51 @@
+package florence.core.model.shared
+
+import florence.core.model.shared.FontLengthUnit.*
+
+type FontSize = FontSizeIn[FontLengthUnit]
```

Comment:
I wonder if we could make this opaque, and hide the `FontSizeIn` type which looks more like something we deal with internally 🤔 
```suggestion
opaque type FontSize <: FontSizeIn[FontLengthUnit] = FontSizeIn[FontLengthUnit]
```

In that case we'd probably need an implicit `Conversion[FontSizeIn[FontLengthUnit], FontSize]` that we can, again, only use internally without exposing to users.

Reply:
I like opaque types, but I'm suspicious of opaque types that sub type the thing they're making opaque, because it removes the opaqueness. Could you just do this, and have explicit helper / extension methods for when conversion is needed?

```scala
opaque type FontSize = FontSizeIn[FontLengthUnit]
```

Reply:
Thanks! let me know if you had in mind something like this: https://github.com/xebia-functional/florence/pull/66/commits/e51b9b767c6a7124fde0605e08b074cbd5205a8f

Reply:
Looks good to me. 👍